pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -229,6 +221,12 @@ clangStdenv.mkDerivation (finalAttrs: {
       "-DUSE_GTK4=OFF"
     ] ++ lib.optionals (!systemdSupport) [
       "-DENABLE_JOURNALD_LOG=OFF"
+    ] ++ lib.optionals (clangStdenv.hostPlatform != clangStdenv.buildPlatform) [
+      # Ignore gettext in cmake_prefix_path so that find_program doesn't
+      # pick up the wrong gettext. TODO: Find a better solution for
+      # this, maybe make cmake not look up executables in
+      # CMAKE_PREFIX_PATH.
```

Comment:
I believe this was done in https://github.com/NixOS/nixpkgs/pull/318226 so perhaps it can simply go.